frontend/javascripts/oxalis/view/right-border-tabs/export_bounding_box_modal.js

```diff
@@ -178,9 +191,7 @@ const ExportBoundingBoxModal = ({ handleClose, dataset, boundingBox, tracing }:
   let activeMappingMessage = null;
   if (isMergerModeEnabled) {
     activeMappingMessage =
-      "Exporting a volume layer does not export merger mode currently. Please disable merger mode before exporting data.";
-  } else if (isMappingEnabled) {
-    activeMappingMessage = `The active mapping ${mappingName} will be applied to the exported data.`;
```

Comment:
Did you remove this because it no longer happens or because the message was superfluous?

Reply:
The message here was shown once (or not at all) for the entire modal. However, when there are multiple segmentation layers, the message needs to appear per layer. That's why, I moved the message into the layer-buttons (see https://github.com/scalableminds/webknossos/pull/5683/files/41a3695bc1e72a95ea8fa849b2fbb90da59d2975#diff-c3b0e8e7fff2777038951ac1d27237b7cf1fc00ea84b72eb977fde66242b7ddaR132).